src/Form/Field/elements.js

```diff
@@ -26,7 +26,14 @@ export const Wrapper = styled.div`
   ${({ isRow }) =>
     isRow &&
     css`
-      margin-left: ${({ theme: { dimensions } }) => dimensions.small};
+      margin-left: ${({ theme: { dimensions } }) => dimensions.big};
+      margin-top: 0;
```

Comment:
`isRow` and `inlineLabel` have a `margin-top` you should avoid duplication and apply style directly to the component